File: BaekJoon/JS/if문/2753.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().split(' ').map(val => +val);
const A = input;

if (A % 4 == 0 && A % 100 != 0) {
    console.log(1)
} else if (A % 400 == 0) {
    console.log(1)
} else {
    console.log(0)
};

File: Python/if문/2753.py
a, b = map(int, input().split())

if b - 45 >= 0:
    a = a
    b = b - 45
else:
    if a - 1 == -1:
        a = 23
        b = 60 - 45 + b
    else:

        a = a - 1
        b = 60 - 45 + b
        
print(a, end=' ') 
print(b)